.github/workflows/pr.yaml

```diff
@@ -161,3 +161,137 @@ jobs:
             -tracee-image-ref "tracee:latest" \
             -tracee-tester-image-ref "aquasec/tracee-tester:latest" \
             -tracee-signatures "TRC-2"
+
+  #
+  # Tests TRC-2 with selected kernels.
+  # Runs on a PR touching eBPF source codes.
+  #
+  # TODO: Automate tracee-test-kernels generation through workflows.
+  # TODO: Change dockerhub -> github repository once we're sure this is good.
+  #
+
+  #
+  # TRC-2 v4.19 (stable)
+  #
+  run-core-test-trc2-v419-stable:
```

Comment:
Instead of duplicating similar job definitions we can use [matrix](https://docs.github.com/en/actions/using-jobs/using-a-matrix-for-your-jobs) strategy:

```yaml
jobs:
  run-core-test:
    runs-on: ubuntu-20.04
    needs:
      - smoke-test-core
    strategy:
      matrix:
        signature:
          - "TRC-2"
        kernel:
          - "4.19.238-stable"
          - "5.4.189-stable"
          - "5.8.18-ubuntu"
          - "5.10.111-stable"
          - "5.13.19-ubuntu"
    steps:
      - name: "Checkout"
        uses: actions/checkout@v2
        with:
          submodules: true
      - name: "Pull test container image"
        run: |
          docker image pull rafaeldtinoco/tracee-test-kernels:latest
      - name: "Run tests"
        run: |
          docker run --rm --privileged -v $(pwd):/tracee:rw \
            -e kvm_accel="tcg" \
            -e kern_version=${{ matrix.kernel }} \
            -e test_name=${{ matrix.signature }} \
            -t rafaeldtinoco/tracee-test-kernels:latest
```